src/sinks/amqp/config.rs

```diff
@@ -18,6 +18,11 @@ pub struct AmqpPropertiesConfig {
     /// Content-Encoding for the AMQP messages.
     #[configurable(derived)]
     pub(crate) content_encoding: Option<String>,
+
+    /// Persistent delivery mode for the AMQP messages.
+    #[serde(default = "crate::serde::default_false")]
+    #[configurable(derived)]
+    pub(crate) persistent: bool,
```

Comment:
Since you added a new option, you also need to update the corresponding documentation: https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sinks/amqp.cue . It can be done with `make generate-component-docs` command - it will regenerate the corresponding documentation.

Reply:
Thanks! Will do.